src/main/java/com/iota/iri/LedgerValidator.java

```diff
@@ -15,6 +15,8 @@
  */
 public class LedgerValidator {
 
+    public static final int MAX_REF_MILESTONE_DEPTH = 15;
```

Comment:
Consider possibly doubling this with the configuration parameter of a similar name.